apps/frontend/src/modules/groups/slices/group.slice.ts

```diff
@@ -23,6 +25,7 @@ const { actions, name, reducer } = createSlice({
 		});
 		builder.addCase(loadAll.fulfilled, (state, action) => {
 			state.groups = action.payload.items;
+			state.totalGroupsCount = action.payload.totalItems;
```

Comment:
the same as for users
```suggestion
			state.groupsTotalCount = action.payload.totalItems;
```

Reply:
I've adjusted this to match users.